app/views/ShareView/Thumbs.tsx

```diff
@@ -85,23 +88,49 @@ const ThumbContent = React.memo(({ item, theme, isShareExtension }) => {
 					<CustomIcon name='camera' size={30} color={themes[theme].tintColor} />
 				</View>
 			);
-		} else {
-			const { uri } = item;
-			return (
-				<>
-					<Image source={{ uri }} style={styles.thumb} />
-					<CustomIcon name='camera-filled' size={20} color={themes[theme].buttonText} style={styles.videoThumbIcon} />
-				</>
-			);
 		}
+		const { uri } = item;
+		return (
+			<>
+				<Image source={{ uri }} style={styles.thumb} />
+				<CustomIcon name='camera-filled' size={20} color={themes[theme].buttonText} style={styles.videoThumbIcon} />
+			</>
+		);
 	}
 
 	// Multiple files upload of files different than image/video is not implemented, so there's no thumb
 	return null;
 });
 
-const Thumb = ({ item, theme, isShareExtension, onPress, onRemove }) => (
-	<ThumbButton style={styles.item} onPress={() => onPress(item)} activeOpacity={0.7}>
+// isIOS ? TouchableOpacity : TouchableNativeFeedback
+
+interface IThumbButton {
+	onPress(): void;
+	children: JSX.Element;
+}
+
+const ThumbButton = ({ onPress, children }: IThumbButton) => {
```

Comment:
As discussed in a meeting, rollback this new component for:

```const ThumbButton: typeof React.Component = isIOS ? TouchableOpacity : TouchableNativeFeedback;```

We'll refactor this code in a opportune time